driving_school/demo/website_view.xml

```diff
@@ -0,0 +1,414 @@
+                          <div class="accordion-collapse collapse" data-bs-parent="#myCollapse" role="region" id="myCollapseTab788126_3" aria-labelledby="accordion-button788126_3">
+                            <div class="accordion-body">
+                              <p>We offer a 24-hour cancellation policy for all lessons. If you need to cancel or reschedule, please notify us at least 24 hours in advance to avoid a cancellation fee.</p>
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </section>
+            <section class="s_cta_box pt80 pb80 o_cc o_cc2" data-snippet="s_cta_box">
+              <div class="container">
+                <div class="s_card o_card_img_horizontal flex-lg-row-reverse card o_cc o_cc1" data-snippet="s_card" data-vxml="001" data-name="Card" style="--card-spacer-x: 64px; --card-spacer-y: 64px; border-width: 0px !important;">
+                  <figure class="o_card_img_wrapper ratio ratio-1x1 mb-0">
+                    <img class="o_card_img rounded-end" src="/web/image/driving_school.configurator_1_s_banner_default_image_2" alt="" loading="lazy"/>
+                  </figure>
+                  <div class="card-body">
+                    <h2 class="card-title">
+        Transform your driving skills with<br/>our lessons. Contact us<br/>to start your journey!<br/>
+    </h2>
+                    <p>
+                      <br/>
+                    </p>
+                    <a class="btn btn-secondary" href="/appointment">Contact Us</a>
+                  </div>
+                </div>
+              </div>
+            </section>
+          </div>
+        </t>
+      </t>
+    </field>
+  </record>
+  <record id="ir_ui_view_3048" model="ir.ui.view">
+    <field name="arch" type="xml">
+      <t t-name="website.pricing-1">
+        <t t-call="website.layout">
+          <div id="wrap" class="oe_structure oe_empty">
+            <section class="s_text_block pt40 pb0 o_colored_level" data-snippet="s_text_block">
+              <div class="container s_allow_columns">
+                <h1 style="text-align: center;">Pricing</h1>
+              </div>
+            </section>
+            <section class="s_text_image pt64 pb64 o_colored_level" data-snippet="s_image_text">
+              <div class="container">
+                <div class="row align-items-center">
+                  <div class="pt16 pb16 col-lg-5 o_colored_level">
+                    <img src="/unsplash/JgEtmgOjoew/1586/driving.jpg?unique=f48e2f33" alt="man driving straight on pathway during day time" class="img img-fluid mx-auto rounded o_we_custom_image" data-mimetype="image/webp" data-original-id="1581" data-original-src="/unsplash/JgEtmgOjoew/driving.jpg?unique=ac2b323c" data-mimetype-before-conversion="image/jpeg" data-resize-width="690"/>
```

Comment:
I heard we may need to stop using unsplash. So maybe worth having this image in the attachment instead